sycl/doc/syclcompat/README.md

```diff
@@ -1729,6 +1729,140 @@ static void invoke_kernel_function(kernel_function &function,
 
 } // namespace syclcompat
 ```
+The kernel_launcher class provides a utility for launching SYCL kernels
+through function wrappers, managing execution configurations such as SYCL
+queues, nd_range, and local memory size as thread-local variables. It allows
+kernel invocation through multiple overloads of launch, either by directly
```

Comment:
```suggestion
kernel invocation through multiple overloads of `launch`, either by directly
```
Also, would it make sense to have a link to the section with the `launch` overloads?